src/scripts/Publish-Toolkit.ps1

```diff
@@ -50,6 +51,12 @@ $repoConfig = @{
         relativePath  = "modules/cost"
         requiredFiles = @("main.bicep", "main.json", "metadata.json", "README.md", "version.json")
     }
+    appInsights = @{
+      mainBranch    = 'master'
+      possibleNames = @($appInsightsRepo, 'Application-Insights-Workbooks')
+      relativePath  = "Workbooks/Azure Advisor/Cost Optimization"
+      requiredFiles = @("CostOptimization.workbook","Storage.workbook","Networking.workbook","Compute.workbook","AHB.workbook","Reservations.workbook")
```

Comment:
Does this work as-is or is this just a placeholder? I'm fine either way. Just curious. Seems too simple to work fully 😜

Reply:
no I'm not yet done, I just started to work on it, there is some logic I need to put there.